Package.swift

```diff
@@ -118,25 +19,29 @@ let package = Package(
         .iOS(.v13)
     ],
     products: [
-        .library(name: "AWSClientRuntime", targets: ["AWSClientRuntime"])
+        .library(name: "AWSClientRuntime", targets: ["AWSClientRuntime"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/awslabs/smithy-swift", .exact("0.3.0")),
+        .package(url: "https://github.com/awslabs/aws-crt-swift", .exact("0.2.2"))
```

Comment:
We are targeting newer versions now
```suggestion
        .package(url: "https://github.com/awslabs/smithy-swift", .exact("0.4.0")),
        .package(url: "https://github.com/awslabs/aws-crt-swift", .exact("0.3.0"))
```

Reply:
Pinned versions have been caught up to the latest release.